bundles/org.openhab.core.model.script/src/org/openhab/core/model/script/actions/Audio.java

```diff
@@ -96,18 +109,15 @@ public static synchronized void playStream(@ParamDoc(name = "sink", text = "the
         }
     }
 
-    @ActionDoc(text = "gets the master volume", returns = "volume as a float in the range [0,1]")
+    @ActionDoc(text = "gets the master volume", returns = "volume as a float in the range [0;1]")
     public static float getMasterVolume() throws IOException {
         return AudioActionService.audioManager.getVolume(null).floatValue() / 100f;
     }
 
     @ActionDoc(text = "sets the master volume")
     public static void setMasterVolume(
-            @ParamDoc(name = "volume", text = "volume in the range [0,1]") final float volume) throws IOException {
-        if (volume < 0 || volume > 1) {
-            throw new IllegalArgumentException("Volume value must be in the range [0,1]!");
-        }
-        setMasterVolume(new PercentType(new BigDecimal(volume * 100f)));
+            @ParamDoc(name = "volume", text = "volume in the range [0;1]") float volume) throws IOException {
```

Comment:
Can you re-insert the `final` for consistency here?

Reply:
Done.